features/netsocket/emac-drivers/TARGET_WHD/interface/WhdSTAInterface.cpp

```diff
@@ -231,7 +234,6 @@ nsapi_error_t WhdSTAInterface::set_credentials(const char *ssid, const char *pas
 
 nsapi_error_t WhdSTAInterface::connect()
 {
```

Comment:
This connect is not protected, but it's public. Is that safe?

Reply:
Agree. connect() has a smaller scope and share underlying resource with SoftAP.